File: cs/Iris/ParserUtils/LexParseException/InvalidSequence.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class InvalidSequence : BaseException
    {
        public InvalidSequence(int lineNum, int colNum)
            : base(lineNum, colNum)
        {

        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append("invalid sequence: ");
            str.Append(base.stringTrace());

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/InvalidSequence.java
package com.rokuan.iris.exception;

public class InvalidSequence extends BaseException
{
	public InvalidSequence(int lineNum, int colNum)
	{
		super(lineNum, colNum);
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append("invalid sequence: ");
		str.append(super.stringTrace());

		return str.toString();
	}
}
